examples/18013-5/sd_jwt_payload.json

```diff
@@ -0,0 +1,38 @@
+{
+  "iss": "https://example.com",
+  "iat": 1678516670,
+  "exp": 1678517570,
+  "cnf": {
+    "jwk": {
+      "kty": "EC",
+      "crv": "P-256",
+      "x": "7jIrJ0veNwJa4p7WoDl9stzlxvgj3X4autz8NkR7vWQ",
+      "y": "cDE-9B9gO6h-OSA033Ua3wfVtHEdl-WBHC3dxNUkoAY"
+    }
+  },
+  "docType": "org.iso.18013.5.1.mDL",

```

Comment:
ok, let's try do to some clarification.

this representation doesn't contain the elements of a typical mdoc structure like the one below

````
{
 'version': '1.0',
 'documents': Array of $doc,
 'status': 0
}
````
but just a doc, and its ok only if we explicitly say it in the text. At the same time we find the claim `version` in the proposed example, so it seems a mix. Just give some clarification about this not conventional data structure.

Going to the single doc (considering that an mdoc is able to contain more than a single doc), we have

````
{
 "docType": "org.iso.18013.5.1.mDL",
 "issuerSigned": {
     "nameSpaces":  {
         "org.iso.18013.5.1": [ $array of digests ]}, 
     "issuerAuth": {CBORTag 18 containing a COSE Sign1 Document without a content-type, that's the MSO}
 },
 "deviceSigned": ... not of interest for SD-JWT
}
````

I agree about the flattened approach proposed in this PR, that makes sd-jwt one step forward.

Probably the nameSpaces may have the `nameSpace` as a dictionary, as it is in mdoc cbor